Models/Security/Redactor.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Text;
+using static Models.Enums.Enums;
+
+namespace Models.Security
+{
+    public class Redactor : EntityBase
+    {
+        [ForeignKey("User")]
+        public long UserId { get; set; }
+
+        public User User { get; set; }
+
```

Comment:
i think that we need to implement redactor from user
public class Redactor : User

Reply:
Vladislav, I think that is bad idea. If we need add or delete some property from User, we should do changes in different tables.